config/initializers/statsd.rb

```diff
@@ -33,7 +33,21 @@ class << Samson
 end
 
 ActiveSupport::Notifications.subscribe("job_status.samson") do |*, payload|
-  Samson.statsd.increment "jobs.#{payload.fetch(:type)}.#{payload.fetch(:status)}"
+  tags = [
+    "project:#{payload.fetch(:project)}",
+    "stage:#{payload.fetch(:stage)}",
+  ]
+  total_latency = payload.fetch(:total_latency)
+  pr_production_latency = total_latency.fetch(:pr_production_latency)
```

Comment:
maybe the structure should be `latencies: {pr_production: 123, staging_productiom: 234}`